oci/main.py

```diff
@@ -26,21 +29,80 @@ def run():
         'change',
         help='Gerrit change number')
 
+    system_tests_parser = subparsers.add_parser(
+        "system-tests",
+        help="system-tests for a change")
+    system_tests_parser.set_defaults(command=system_tests)
+
+    system_tests_parser.add_argument(
+        'change',
+        help='Gerrit change number')
+
+    system_tests_parser.add_argument(
+        '--engine_version',
```

Comment:
Options should use --, not _. argparse convert --system-tests to system_test automatically.

Reply:
Done